### ISSUE TITLE
URLs rendering in toots may be improved

### ISSUE THREAD

**Issue by AMDmi3** [mastodon/mastodon#6351]
In a toot, URL is simplified by stripping schema, so `http://openports.se` is rendered as `openports.se`. This seem to be good for readability.

However this lacks stripping trailing slash after the domain name: `http://openports.se/` is rendered as `openports.se/`, while it should look like `openports.se` which is more readable.

Also, the markup used for link seem to be unoptimal. Firefox inspector shows the following code:

```
<a href="http://openports.se/" rel="noopener" target="_blank" class="status-link" title="http://openports.se/"><span class="invisible">http://</span><span class="">openports.se/</span><span class="invisible"></span></a>
```

Notice the empty `<span>` at the end.

* * * *

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by jh97uk** [comment]
I think this may have been fixed since? On `v4.3.0-alpha.0`, it appears fixed (at leas the stripping of the trailing slash).

**Comment by AMDmi3** [comment]
6 years later, I do not consider this a bug, as `/` is a relevant part of an URL like any other path. Still could be a probably non-harmful cosmetic change to strip it, but I no longer suggest to. Let me close this as not planned, since no commits are linked to this issue it it could'be be fixed independently. Not sure where to check 4.3.0-alpha, but 4.2.4 which my instance currently runs still doesn't strip the slash.
